<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>594</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/newPrefix/images/Play.png</normaloff>:/newPrefix/images/Play.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QPushButton" name="playButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/newPrefix/images/Play.png</normaloff>:/newPrefix/images/Play.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="stopButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/newPrefix/images/Stop.png</normaloff>:/newPrefix/images/Stop.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="4">
     <widget class="QPushButton" name="pauseButton">
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="resources.qrc">
        <normaloff>:/newPrefix/images/Pause.png</normaloff>:/newPrefix/images/Pause.png</iconset>
      </property>
     </widget>
    </item>
    <item row="0" column="5">
     <widget class="Phonon::SeekSlider" name="seekSlider" native="true">
      <property name="minimumSize">
       <size>
        <width>301</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>16777215</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="8">
     <widget class="QLabel" name="totalLabel">
      <property name="text">
       <string>00:00:00</string>
      </property>
     </widget>
    </item>
    <item row="0" column="7">
     <widget class="QLabel" name="dividerLabel">
      <property name="text">
       <string>/</string>
      </property>
     </widget>
    </item>
    <item row="0" column="6">
     <widget class="QLabel" name="timeLabel">
      <property name="text">
       <string>00:00:00</string>
      </property>
     </widget>
    </item>
    <item row="1" column="5">
     <widget class="Phonon::VolumeSlider" name="volumeSlider" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>50</width>
        <height>0</height>
       </size>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>594</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_Open">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="openAction"/>
    <addaction name="openFolderAction"/>
    <addaction name="exitAction"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="infoAction"/>
    <addaction name="aboutAction"/>
   </widget>
   <addaction name="menu_Open"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="openAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/newPrefix/images/Add.png</normaloff>:/newPrefix/images/Add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="openFolderAction">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Open &amp;Folder</string>
   </property>
   <property name="visible">
    <bool>false</bool>
   </property>
  </action>
  <action name="exitAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/newPrefix/images/Close.png</normaloff>:/newPrefix/images/Close.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
  </action>
  <action name="aboutAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/newPrefix/images/Help.png</normaloff>:/newPrefix/images/Help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
  </action>
  <action name="infoAction">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/newPrefix/images/Info.png</normaloff>:/newPrefix/images/Info.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Info</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
